Code/Tools/ProjectManager/Source/CreateAGemScreen.h

```diff
@@ -40,13 +39,17 @@ namespace O3DE::ProjectManager
     private slots:
         void HandleBackButton();
         void HandleNextButton();
-        void UpdateNextButtonToCreate();
+        void HandleGemTemplateSelectionTab();
+        void HandleGemDetailsTab();
+        void HandleGemCreatorDetailsTab();
 
     private:
         void LoadButtonsFromGemTemplatePaths(QVBoxLayout* gemSetupLayout);
         QScrollArea* CreateGemSetupScrollArea();
         QScrollArea* CreateGemDetailsScrollArea();
         QScrollArea* CreateGemCreatorScrollArea();
+        QFrame* CreateLHSTabs();
+        QFrame* CreateRHSPane();
```

Comment:
I'm assuming LHS and RHS stand for LeftHandSide and RightHandSide?  Other suggestions are `CreateTabButtonsFrame` and `CreateTabPaneFrame`

Reply:
that is correct. I can change it to the suggested names if that makes the code clearer

Reply:
LHS and RHS is a term used commonly for designers. I appreciate you integrated it into the code, but it may not be widely know for others outside of designing.

Reply:
That's a fair point. And architecturally speaking we may choose to change the layout later on, so Alex's suggestion will probably age better long term.